tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "target": "ES2016",
     "allowJs": true,
     "sourceMap": true,
-    "types": [],
+    "types": ["sweetalert2"],
```

Comment:
Still not sure I understand why this had to be added. Can't you just add the typings package for sweetalert2?

Reply:
Tbh I don't understand it either...
I just know that it fails to find the type as the imported file is not a "module" itself but a normal js file when I don't add it.

A separate typings package does not exist (if you meant sth like npm install @types/sweetalert2). It is part of the sweetalert2 package but does not get recognized when I don't add this "types" thingy. (I'm no node pro, so please tell me if you have any other ideas.)

Importing the sweetalert2 module (e.g `import Swal from 'sweetalert2'`) leads to it auto injecting the CSS file which breaks the style overrides.
One would need to add a bunch of strategic !important statements to the custom styling CSS which seemed excessive to me.